### PR TITLE
[2.5] AAP-30807-A: Add General inventory variables for container (#1994)

### PR DESCRIPTION
**Pull request by lmalivert**
- Backport for [PR#1994](https://github.com/ansible/aap-docs/pull/1994)

- [RPM Installation 2.5 Draft](https://docs.google.com/document/d/1fIrlSrj8bSSWBVptXWmm0ZFixhZ4mM5pW2CQx_3Oh7A/edit#heading=h.pcplsjwxwvz8)

- Used this[ Inventory var diff](https://docs.google.com/spreadsheets/d/1wSZiIoDu-_uRY8l1R1vYQlOG5PPrK4MrP3jla8701KA/edit?gid=413143671#gid=413143671) spreadsheet to include container variables to the table: [General variables](https://docs.google.com/document/d/1fIrlSrj8bSSWBVptXWmm0ZFixhZ4mM5pW2CQx_3Oh7A/edit#heading=h.pcplsjwxwvz8)

- Made a revision to the assembly to reorder the tables.

### CODE REVIEW

File: downstream/modules/platform/ref-general-inventory-variables.adoc [hunk 1]
```diff
@@ -21,12 +21,12 @@ The TLSv1.3 parameter only works when OpenSSL 1.1.1 or higher is used.
...
 
 Default = `TLSv1.2`.
-| *`nginx_user_http_config`* | List of nginx configurations for `/etc/nginx/nginx.conf` under the http section. 
+| *`nginx_user_http_config`* | | List of nginx configurations for `/etc/nginx/nginx.conf` under the http section. 
 
 Each element in the list is provided into `http nginx config` as a separate line. 
 
 Default = empty list
-| *`registry_password`* | `registry_password` is only required if a non-bundle installer is used.
+| *`registry_password`* |`registry_password` |`registry_password` is only required if a non-bundle installer is used.
```
Comment (g-murray) [inline downstream/modules/platform/ref-general-inventory-variables.adoc:29]: Just a suggestion but I find it reads better if you include the article here

"The `registry_password` is required if a non-bundle installer is used"

File: downstream/modules/platform/ref-general-inventory-variables.adoc [hunk 2]
```diff
@@ -35,24 +35,56 @@ Enter your Red Hat Registry Service Account credentials in `registry_username` a
 When `registry_url` is `registry.redhat.io`, username and password are required if not using a bundle installer.
```
Comment (g-murray) [inline downstream/modules/platform/ref-general-inventory-variables.adoc:35]: Similar to the above comment "the/your username and password are required if not using a bundle installer."

File: downstream/modules/platform/ref-general-inventory-variables.adoc [hunk 3]
```diff
@@ -35,24 +35,56 @@ Enter your Red Hat Registry Service Account credentials in `registry_username` a
...
 
 For more information, see xref:proc-set-registry-username-password[Setting registry_username and registry_password].
-| *`registry_url`* | Default = `registry.redhat.io`.
-| *`registry_username`* | `registry_username` is only required if a non-bundle installer is used.
+| *`registry_url`* |`registry_url` | URL for the registry source. 
+
+Default = `registry.redhat.io`.
+| *`registry_username`* |`registry_username` |`registry_username` is only required if a non-bundle installer is used.
```
Comment (g-murray) [inline downstream/modules/platform/ref-general-inventory-variables.adoc:41]: Same as above 

""The registry_password is required if a non-bundle installer is used"

**Review by g-murray — COMMENTED** [review]
Hey Lydie, a few optional comments to apply if you agreed with them. 

Also, I tried to preview this locally and got the following error and your edits didn't show in the build (But I did preview ok in VSC).

 I couldn't figure out where exactly in the doc it was referring too. 

"Opening and ending tag mismatch: literal line 2031 and phrase, line 2031, column 169 (line 2031)
Unable to parse the AsciiDoc built DocBook XML"

**Review by michellemacrh — APPROVED** [review]
LGTM

**Pull request merged**